src/main/java/com/example/onlinebookstore/controller/BookController.java

```diff
@@ -34,7 +35,7 @@ public BookDto getBookById(@PathVariable Long id) {
 
     @ResponseStatus(HttpStatus.CREATED)
     @PostMapping
-    public BookDto createBook(@RequestBody CreateBookRequestDto bookDto) {
+    public BookDto createBook(@RequestBody @Valid CreateBookRequestDto bookDto) {
         return bookService.save(bookDto);
     }
 
```

Comment:
if you have update method add also valid annotation there

Reply:
done